src/Infrastructure/TransactionMultisig.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+
+namespace evias\NEMBlockchain\Infrastructure;
+
+use evias\NEMBlockchain\NemSDK;
+use evias\NEMBlockchain\Models\Transaction\Multisig;
+use evias\NEMBlockchain\Models\Transaction\Transfer;
+use evias\NEMBlockchain\Models\Transaction\Mosaic;
+
+
+
+class TransactionMultisig {
```

Comment:
why the need for a specific class for Multisig transactions class and another Multisig class? I believe it is best to keep it all in the Transaction class and let the transaction class handle wrapping like this class here handles it.

In Transaction::multisig() you would wrap the transaction in a "otherTrans" attribute. I think handling multisigs as *specific objects* is wrong as it will also split fees definition etc.. about transactions in several classes making it too complicated to work with.